game-core/src/main/java/games/strategy/engine/data/ResourceCollection.java

```diff
@@ -153,7 +153,7 @@ public int fitsHowOften(final IntegerMap<Resource> cost) {
         resources.subtract(cost);
       } catch (final IllegalArgumentException iae) {
         // when the subtraction isn't possible it will throw an exception, which means we can return
-        // i;
+        // i
```

Comment:
This looks super weird, could you move parts of the previous line here, so that the I isn't lonely

Reply:
Sure: be95fc8b393